testdata/src/main/java/com/hedvig/app/testdata/feature/home/Data.kt

```diff
@@ -39,11 +40,11 @@ val HOME_DATA_TERMINATED_WITH_ONGOING_CLAIMS =
         withClaimStatusCards = true
     ).build()
 
-val HOME_DATA_PENDING_NO_FIRST_NAME =
-    HomeDataBuilder(listOf(ContractStatus.PENDING), firstName = null).build()
-
 val HOME_DATA_ACTIVE = HomeDataBuilder(listOf(ContractStatus.ACTIVE)).build()
 
+val HOME_DATA_CONNECT_PAYIN =
```

Comment:
Maybe change the name to be a bit more representative of the fact that the paying is specifically "NEEDS_SETUP"? Otherwise on the call-site it might look non-obvious about what it does